design/crud-api/cmd/server/service.go

```diff
@@ -305,6 +306,16 @@ func main() {
 		Password: os.Getenv("NEO4J_PASSWORD"),
 	}
 
+	// Initialize PostgreSQL config
+	postgresConfig := &postgres.Config{
+		Host:     os.Getenv("POSTGRES_HOST"),
+		Port:     5432, // Default PostgreSQL port
```

Comment:
@Isuru-rangana shall we take this from env variable as well? Because we may need this configurable in future. WDYT?